.vscode/settings.json

```diff
@@ -63,6 +63,85 @@
     "xstring": "cpp",
     "xutility": "cpp",
     "iosfwd": "cpp",
-    "sstream": "cpp"
+    "sstream": "cpp",
```

Comment:
You may want to try a plugin like [this](https://marketplace.visualstudio.com/items?itemName=chrislajoie.vscode-modelines) which can read modelines, rather than trying to keep a list of headers. I can't vouch for this specific plugin, but I use [something similar](https://github.com/sfiera/vim-emacsmodeline) in vim to deal with this and it works well.

Reply:
I am unsure if this will help. This is list is kept by VS Code for some reason and is constantly updated. I suspect it is the cmake extension that does this. Its not something we are doing manually.